global/scripts/aff2rigid_world

```diff
@@ -0,0 +1,138 @@
+#   further disclaims any liability for the outcomes arising from using
+#   the Software.
+#
+#   The Licensee agrees to indemnify the University and hold the
+#   University harmless from and against any and all claims, damages and
+#   liabilities asserted by third parties (including claims for
+#   negligence) which arise directly or indirectly from the use of the
+#   Software or the sale of any products based on the Software.
+#
+#   No part of the Software may be reproduced, modified, transmitted or
+#   transferred in any form or by any means, electronic or mechanical,
+#   without the express permission of the University. The permission of
+#   the University is not required if the said reproduction, modification,
+#   transmission or transference is done without financial return, the
+#   conditions of this Licence are imposed upon the receiver of the
+#   product, and all original and amended source code is included in any
+#   transmitted product. You may be held legally responsible for any
+#   copyright infringement that is caused or encouraged by your failure to
+#   abide by these terms and conditions.
+#
+#   You are not permitted under this Licence to use this Software
+#   commercially. Use for which any financial return is received shall be
+#   defined as commercial use, and includes (1) integration of all or part
+#   of the source code or the Software into a product for sale or license
+#   by or on behalf of Licensee to third parties or (2) use of the
+#   Software or any derivative of it for research with the final aim of
+#   developing software products for sale or license to a third party or
+#   (3) use of the Software or any derivative of it for research with the
+#   final aim of developing non-software products for sale or license to a
+#   third party, or (4) use of the Software to provide any service to an
+#   external organisation for which payment is received. If you are
+#   interested in using the Software commercially, please contact Oxford
+#   University Innovation ("OUI"), the technology transfer company of the
+#   University, to negotiate a licence. Contact details are:
+#   fsl@innovation.ox.ac.uk quoting Reference Project 9564, FSL.
+
+from __future__ import print_function
+import sys
+from sys import argv
+from numpy import *
+
+def usage():
+    print("Usage: " + argv[0] + " <input2standard mat> <output mat>")
+    print(" ")
+    print("       First argument is the FLIRT transform (12 DOF) from the input image to standard")
+    print("       Second argument is the output matrix which will go from the input image to standard space (6 DOF)")
+    print("          aligning the AC, the AC-PC line and the mid-sagittal plane (in order of decreasing accuracy)")
+    sys.exit(1)
+
+if len(argv) < 2:
+    usage()
+
+# Load in the necessary info
+a=loadtxt(argv[1])
+# set specific AC and PC coordinates in FLIRT convention (x1=AC, x2=PC, x3=point above x1 in the mid-sag plane)
```

Comment:
Change this comment to say something like "AC and PC coordinates in nifti mm convention".

Reply:
I edited it by clicking "view changes" and "edit file" and commited. I hope it is properly modified.

Reply:
Updating the usage was also a good idea, which I should have suggested.  However, I think the comments immediately above the `x1`, `x2`, and `x3` coordinates should be changed so that they don't claim to be "FLIRT convention", because they aren't anymore.

Reply:
Sorry, I changed that part.